### PR TITLE
Rebuild of one-pager

### PR DESCRIPTION
**Pull request by kyleshepherd**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 _site
+.idea
```
Comment (darylldoyle) [inline .gitignore]: 🙌

File: assets/css/input.scss [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+  .ti-img {
+    position: relative;
+
+    .ti-img-bg {
+      position: absolute;
+      top: 2em;
+      bottom: -2em;
+      left: -2em;
+      right: 0;
+      background: $darkcoral url("/assets/images/computer.png") repeat-x center;
```
Comment (darylldoyle) [inline assets/css/input.scss:108]: Any chance we can get this as an SVG? Just looks nicer and is lighter in weight 🙂

File: index.html [hunk 3]
```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+<html>
+    <head>
+        <title>PHP Cornwall</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="PHP Cornwall Community Group">
+        <meta name="keywords" content="PHP, Wordpress, Cornwall, Web Dev">
+        <meta name="theme-color" content="#dee4ff">
+        <link rel="stylesheet" type="text/css" href="assets/css/output.css">
+        <link rel="stylesheet" type="text/css" href="assets/css/normalize.css">
```
Comment (darylldoyle) [inline index.html:9]: We should be loading normalize before our custom CSS otherwise some of it could be overwritten 😬

File: index.html [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        <main>
+            <section class="text-image-container">
+                <div class="ti-text">
+                    <p>PHP Cornwall is a new user group, aimed at bringing together the extensive PHP community in Cornwall.</p>
+                    <h2>The First meeting</h2>
+                    <p>PHP Cornwall is so new, there hasn't even been a meeting yet!
+                        <br><br>
+                        We'd like to talk to you, the developers of Cornwall and find out exactly what you want from a user group.
+                        <br><br>
+                        For now, the easiest place to get hold of us is on <a href="https://twitter.com/cornwallphp" target="_blank">Twitter</a></p>
```
Comment (darylldoyle) [inline index.html]: When using `target="_blank"` we need to think of the security implications. Take a look at [this link](https://mathiasbynens.github.io/rel-noopener/) for some more info on this.

**Review by darylldoyle — CHANGES_REQUESTED** [review]
We should add `.DS_Store` files to our `.gitignore` really as they're macOS specific and will only lead to unnecessary changes in the repo otherwise.

Also, not too sure where the `ti-bg.jpg` file is being used? I can't see it in the CSS?

On the plus side, great use of semantic HTML tags (header/main/section etc). 
Looks like you're enjoying SCSS too? That said, might be worth dropping a note in the `README.md` about how you're compiling the CSS so others can follow suit?

File: index.html (hunk 3 above)
Comment (kyleshepherd) [reply index.html:9]: Ah okay, moved `normalize` before the custom CSS now

File: index.html (hunk 4 above)
Comment (kyleshepherd) [reply index.html]: Ah right, so I've added `rel="noopener"` to these links, that'll make them more secure right?

File: assets/css/input.scss (hunk 2 above)
Comment (kyleshepherd) [reply assets/css/input.scss:108]: Ah only available as an SVG annoyingly :/

File: .gitignore [hunk 5]
```diff
@@ -1 +1,3 @@
...
+.DS_STORE
```
Comment (darylldoyle) [inline .gitignore:3]: 👍

File: index.html (hunk 4 above)
Comment (darylldoyle) [reply index.html]: Exactly, make sure you read that page to understand the reasoning behind it though. It's a very good example of how it can be used!

Comment (kyleshepherd) [reply index.html]: Nice, okay, I'll have a read!

**Review by darylldoyle — COMMENTED** [review]
👍 Nice

**Review by darylldoyle — APPROVED** [review]

**Pull request merged**